### PR TITLE
CI: Run fuzzer daily

### PR DESCRIPTION
**Pull request by tcharding**
Update the `fuzz/generate-files.sh` script to mimic that in `rust-bitcoin` and generate a new fuzz job that runs the fuzz tests daily for an hour.

Run the fuzz job the time that `rust-bitcoin` fuzz job finishes - I don't know if that matters. I still don't know if there are org wide limits or just repo limits but it doesn't hurt to run at a different time.

### CODE REVIEW

File: fuzz/generate-files.sh [hunk 1]
```diff
@@ -40,13 +41,13 @@ done
...
-    branches:
-      - master
-      - 'test-ci/**'
-  pull_request:
+  schedule:
+    # 5am every day UTC, this correlates to:
+    # - 11pm PDT
+    # - 7am CET
+    # - 5pm AEDT
+    - cron: '00 06 * * *'
```
Comment (storopoli) [inline fuzz/generate-files.sh:50]: This is 6AM right?

**Review by storopoli — APPROVED** [review]
ACK 44ddbe3484029bf42c33928ac1dd7d9a5c8cee84

**Review by apoelstra — APPROVED** [review]
ACK 44ddbe3484029bf42c33928ac1dd7d9a5c8cee84

File: fuzz/generate-files.sh (hunk 1 above)
Comment (storopoli) [reply fuzz/generate-files.sh:50]: We need to change this to `05` or update the comment above.
I know that this is "copy-pasta" left over from the another PR that I saw earlier that had both `06` and `6am`

Comment (apoelstra) [reply fuzz/generate-files.sh:50]: Yep. (At least, that's how I read it.)

Comment (tcharding) [reply fuzz/generate-files.sh:50]: Oh woops, I changed the three bottom ones and missed the top one (not shown here). While investigating I noticed that I totally botched this PR, I didn't change the output file to `cron-fuzz-daily.yml` and I didn't run the script either - god damn, it is hard to get good help. Will follow up.

Comment (tcharding) [reply fuzz/generate-files.sh:50]: Man, now I'm confused. A job cron-daily-fuzz ran last night.

Comment (tcharding) [reply fuzz/generate-files.sh:50]: Oh, I must have created the yaml file manually and not run `generate-files.sh` - face palm.

Comment (tcharding) [reply fuzz/generate-files.sh:50]: https://github.com/rust-bitcoin/rust-miniscript/pull/684

Thanks for the careful review @storopoli!

Comment (apoelstra) [reply fuzz/generate-files.sh:50]: I'm also confused -- I ran the script locally and the diff was clean (didn't even create any other files). Are you saying it did nothing?

Comment (tcharding) [reply fuzz/generate-files.sh:50]: Seems its too early in the morning for my brain to context switch back into this but if you check `cron-daily-fuzz.sh` you will ask yourself "who wrote this rubbish and how did it get in master?" For example it still runs on `pull_request` - all fixed in #684. As for postmortem I'm not sure how I botched it so badly.

Comment (apoelstra) [reply fuzz/generate-files.sh:50]: I think I must have tested the fuzz script in a different repo or something. This is definitely broken as hell :).

**Pull request merged**